cdk/lib/auth-stack.ts

```diff
@@ -48,6 +50,17 @@ export class AuthStack extends cdk.Stack {
       },
     });
 
+    if (process.env.COGNITO_EMAIL) {
+      new cognito.CfnUserPoolUser(this, "UserPoolUser", {
+        userPoolId: pool.userPoolId,
+        username: process.env.COGNITO_EMAIL.split("@")[0],
+        userAttributes: [
+          { name: "email", value: process.env.COGNITO_EMAIL },
+          { name: "custom:roles", value: JSON.stringify(["Admin"]) },
+        ],
+      });
+    }
```

Comment:
This approach works well with CDK deployments, but we need to make this work with CloudFormation parameters. Customers will be deploying Badger using a CloudFormation template and environment variables is not something they can provide. 

We can do something like this: 
```
const adminEmail = new CfnParameter(this, "adminEmail");

if (adminEmail) {
  new cognito.CfnUserPoolUser(this, "UserPoolUser", {
          userPoolId: pool.userPoolId,
          username: adminEmail,
          userAttributes: [
            { name: "email", value: adminEmail },
            { name: "custom:roles", value: JSON.stringify(["Admin"]) },
          ],
  });
}
```